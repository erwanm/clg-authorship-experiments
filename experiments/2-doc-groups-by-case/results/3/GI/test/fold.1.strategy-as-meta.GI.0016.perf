0.363552	0.649200	0.560000
