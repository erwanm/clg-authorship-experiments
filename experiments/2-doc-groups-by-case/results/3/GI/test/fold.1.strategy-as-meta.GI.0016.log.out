apply-config.sh: applying config '/tmp/expe2/3/process/selected-meta-configs/fold.1.strategy-as-meta.GI.0016' to cases '/home/moreaue/dump/expe2//results/3/GI/test/only-cases.tmp.Y2cRclip'; results written to '/home/moreaue/dump/expe2//results/3/GI/test/fold.1.strategy-as-meta.GI.0016'
apply-config.sh: done.
